pkg/subctl/cmd/gather.go

```diff
@@ -102,6 +102,8 @@ func gatherData() {
 		directory = "submariner-" + time.Now().UTC().Format("20060102150405") // submariner-YYYYMMDDHHMMSS
 	}
 
+	fmt.Printf("Files will be stored under directory %q\n", directory)
```

Comment:
Would be better to have this at the end with a msg like `Files stored in directory xyz`. Gather output can get very long with multiple clusters and it is always best to have user relevant output at the end so user doesn't need to scroll back to look it up.

Reply:
I vote for the end.

As long as our error control is tight and we won't stop mid-flight.